es-ES/mail/chrome/messenger/messenger.properties

```diff
@@ -424,10 +424,12 @@ editVirtualFolderPropertiesTitle=Editar las propiedades de la búsqueda %S
 #alert to inform the user to choose one or more folders to search for a saved search folder
 alertNoSearchFoldersSelected=Debe elegir al menos una carpeta en la que buscar para guardar la búsqueda.
 
-# These two are displayed in the message and folder pane windows
-# LOCALIZATION NOTE(????ByteAbbreviation): Do not translate %d below, it is the size of the message/folder
-kiloByteAbbreviation=%dKB
-megaByteAbbreviation=%dMB
+# These are displayed in the message and folder pane windows
+# LOCALIZATION NOTE %.*f is the abbreviated size in the appropriate units
+byteAbbreviation2=%.*f bytes
```

Comment:
You don't want to replace these, you just want to add the Abbreviation2 versions. Removing the kilo/mega will break other places.

We're also missing giga

https://github.com/postbox/postbox/blob/4f3935c82a440eb6b622bac2015027f309afc192/mail/locales/en-US/chrome/messenger/messenger.properties#L443

We should match  English